### PR TITLE
initial commit for EKS Go vulnerability management tracking

### PR DESCRIPTION
**Pull request by danbudris**
*Issue #, if available:*

*Description of changes:*
Initial commit adding a baisc VEX (vulnerability eXchange) document for EKS Go (see https://github.com/oasis-tcs/csaf/blob/master/csaf_2.0/prose/csaf-v2-editor-draft.md). We can use this to track what CVEs are fixed where and then filter them out of GoVulnCheck etc

Only have one CVE in here right now will add more once this is looking good

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+              "product_id": "eks-distro-golang:v1-19-12-eks-9",
+              "name": "Amazon EKS Distribution Golang version v1.19.12 EKS Release 9"
+            }
+          },
+          {
+            "category": "product_version",
+            "name": "v1.20.7-eks-8",
+            "product": {
+              "product_id": "eks-distro-golang:v1-20-7-eks-8",
+              "name": "Amazon EKS Distribution Golang version v1.20.7 release 8"
```
Comment (danbudris) [inline projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json:62]:
```suggestion
              "name": "Amazon EKS Distribution Golang version v1.20.7 EKS release 8"
```

File: projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+              "product_id": "eks-distro-golang:v1-20-7-eks-8",
+              "name": "Amazon EKS Distribution Golang version v1.20.7 EKS release 8"
+            }
+          },
+          {
+            "category": "product_version",
+            "name": "v1.21.0-eks-0",
+            "product": {
+              "product_id": "eks-distro-golang:v1-21-0",
+              "name": "Amazon EKS Distribution Golang version v1.21.0 release 0"
```
Comment (danbudris) [inline projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json:70]:
```suggestion
              "name": "Amazon EKS Distribution Golang version v1.21.0 EKS release 0"
```

File: projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    ],
+    "publisher": {
+      "category": "vendor",
+      "name": "Amazon Web Services Inc",
+      "namespace": "https://aws.amazon.com/"
+    },
+    "title": "Vulnerability Tracking Document for AWS EKS Distro Golang",
+    "tracking": {
+      "current_release_date": "2023-08-28T11:00:00Z",
+      "id": "2023-08-EKS-D-GO",
```
Comment (markapruett) [inline projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json:20]: What's the significance of this string?

**Review by markapruett — APPROVED** [review]

File: projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json (hunk 3 above)
Comment (danbudris) [reply projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json:20]: tbh i didn't really think about it; i just made the ID yyyy-dd and then our own identifier. suggestions welcome

File: projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+                    "file_hashes": [
+                      {
+                        "algorithm": "",
+                        "value": ""
+                      }
+                    ],
+                    "filename": ""
+                  }
+                }
+          },
```
Comment (danbudris) [inline projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json:59]:
```suggestion
          }
```

**Review by jaxesn — APPROVED** [review]

**Pull request merged**